semgrep-core/matching/Generic_vs_generic.ml

```diff
@@ -1846,7 +1846,7 @@ and m_entity a b =
   { A. name = a1; attrs = a2; tparams = a4; info = a5 },
   { B. name = b1; attrs = b2; tparams = b4; info = b5 } ->
     m_ident_and_id_info_add_in_env_Expr (a1, a5) (b1, b5) >>= (fun () ->
-    (m_list__m_attribute) a2 b2 >>= (fun () ->
+    (m_list_in_any_order ~less_is_ok:true m_attribute a2 b2) >>= (fun () ->
```

Comment:
better to remove the body of m_list__m_attribute and call 
m_list_in_any_order ... in it that way all the places where we look for attributes will have this new behavior.
(need to check the possible perf regression when applied to all the instances).